packages/shorebird_cli/lib/src/commands/release/release_android_command.dart

```diff
@@ -284,16 +286,27 @@ ${summary.join('\n')}
           releaseId: release.id,
           projectRoot: projectRoot.path,
           aabPath: bundlePath,
-          platform: platform,
+          platform: releasePlatform,
           architectures: architectures,
           flavor: flavor,
         );
 
         await codePushClientWrapper.updateReleaseStatus(
           appId: app.appId,
           releaseId: release.id,
-          platform: platform,
+          platform: releasePlatform,
           status: ReleaseStatus.active,
+          metadata: UpdateReleaseMetadata(
+            releasePlatform: releasePlatform,
+            flutterVersionOverride: flutterVersion,
+            generatedApks: generateApk,
+            environment: BuildEnvironmentMetadata(
+              operatingSystem: platform.operatingSystem,
+              operatingSystemVersion: platform.operatingSystemVersion,
+              shorebirdVersion: packageVersion,
+              xcodeVersion: null,
```

Comment:
I suspect we'll want to add AndroidStudio version as well?

Reply:
Yep, although we don't need it immediately. Will file an issue to add.